test/test_mapper_llvm.cc

```diff
@@ -61,6 +66,29 @@ def fun(float(N, M) A, float(N, M) B) -> (C) {
   checkRtol(Cc - C, {A, B}, N * M);
 }
 
+TEST(LLVMCodegen, DISABLED_BasicExecutionEngine) {
```

Comment:
why is this test disabled?

Reply:
This is a skeleton for a test. It just tests that compilation for CPU only and GPU only passes.
The CPU mapper is not yet ready.

Depending on urgency to merge in dev I can continue working on this WIP PR or continue in follow up diffs.

The tradeoff here is really to unblock @salexspb, I am fine with either :)

Reply:
makes sense, shall we create a TODO task for follow-up work and add the task number here? :)

Reply:
Filed as #148 thanks for keeping me honest ;)